controllers/groupController.js

```diff
@@ -0,0 +1,54 @@
+import ApiError from '../error/ApiError';
+import { createGroupService, getGroupsService, getGroupByIdService, updateGroupService, deleteGroupService } from '../services/groupService';
+
+export default class GroupController {
+    async createGroup(req, res) {
+        const createdGroup = await createGroupService(req.body);
+
+        if (createdGroup.errorCode === 409) {

```

Comment:
You are returning `return ApiError.badRequest(error.errors[0].message);` badRequest error from service. Is it possible to have `409` error code?
However it is better to throw errors not to return.

Reply:
fixed